src/en/ss14-by-example/ui-and-you.md

```diff
@@ -0,0 +1,474 @@
+associated with.
+
+-   You should prefer to select elements with `.Identifier` and not `.Class`.
+-   Any styles that COULD be used by another UI should be moved to a generic sheetlet.
+-   Hardcoding is more relaxed, you should still try and avoid it when possible, but
+    hardcoding `StyleClass`es and `StyleIdentifier`s is probably fine.
+-   If you REALLY need a specific resource and its not worth adding it to an
+    `ISheetletConfig` you can access it through `ResCache` as normal.
+-   You don't need to do the generic type constraints thing since the sheetlet
+    should be specific to a single UI, and thus a single stylesheet.
+
+<details>
+<summary>Example Code (click to expand)</summary>
+
+```cs
+using Content.Client.Resources;
+using Content.Client.Stylesheets.Redux;
+using Content.Client.Stylesheets.Redux.SheetletConfigs;
+using Content.Client.Stylesheets.Redux.Stylesheets;
+using Robust.Client.Graphics;
+using Robust.Client.UserInterface;
+using Robust.Client.UserInterface.Controls;
+// you need to add this line manually to access the helper methods
+using static Content.Client.Stylesheets.Redux.StylesheetHelpers;
+
+namespace Content.Client.Paper.UI;
+
+// MAKE SURE TO INCLUDE THE [CommonSheetlet] ATTRIBUTE
+[CommonSheetlet]
+// which stylesheet is this sheetlet for
+public sealed class PaperSheetlet : Sheetlet<NanotrasenStylesheet>
+{
+    public override StyleRule[] GetRules(NanotrasenStylesheet sheet, object config)
+    {
+        // define any IConfigs you need here
+        IWindowConfig windowCfg = sheet;
+
+        // get any textures / construct any complicated resources here
+        var paperBackground = ResCache.GetTexture("/Textures/Interface/Paper/paper_background_default.svg.96dpi.png")
+            .IntoPatch(StyleBox.Margin.All, 16);
+        var paperBox = new StyleBoxTexture
+            { Texture = sheet.GetTexture(windowCfg.TransparentWindowBackgroundBorderedPath) };
+        paperBox.SetPatchMargin(StyleBox.Margin.All, 2);
+
+        // and finally, define all the style rules and return a big 'ol list of them
+        return
+        [
+            E<PanelContainer>().Identifier("PaperContainer").Panel(paperBox),
+            E<PanelContainer>()
+                .Identifier("PaperDefaultBorder")
+                .Prop(PanelContainer.StylePropertyPanel, paperBackground),
+        ];
+    }
+}
+```
+
+</details>
+
+### Making your own `Stylesheet`
+
+Creating a new stylesheet is pretty simple, so I won't go into it too much.
+One thing to keep in mind is how colors are chosen.
+
+The colors are generated using the [OKLAB color space](https://bottosson.github.io/posts/oklab/),
+which is better because something something human eyes something. When you choose
+new colors for your stylesheet, it may be helpful to use an [OKLCH Color Picker](https://oklch.com)
+and modify an existing color.
+
+## Writing C# for UI
+
+> **TODO:** I don't feel confident enough in my knowledge to describe in detail
```

Comment:
I think talking about actual UI implementation can be done in a different page, and you could definitely get away with not having to write about it in this PR if you want :P